dashboard/modules/runtime_env/runtime_env_agent.py

```diff
@@ -59,6 +62,10 @@ class CreatedEnvResult:
     # If success is True, will be a serialized RuntimeEnvContext
     # If success is False, will be an error message.
     result: str
+    # Creation time in milliseconds.
+    creation_time_ms: float
+    # Total number of retries before it is created.
+    retry_cnt: int
```

Comment:
Wonder if we should rename this better

Reply:
max_num_retries

Reply:
@scv119 I think this is for the number of times it was actually retried.

@rkooo567 I would suggest we leave this out until we finalize the retry behavior, I'm not sure we actually want to retry at this layer.

Reply:
Sounds good to me. We will also have internal API review and fill up missing information in the follow-up PRs. I will leave it for now.